src/Microsoft.AspNetCore.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -286,41 +293,44 @@ public void WriteTo(TextWriter writer, HtmlEncoder encoder)
                 writer.Write("<");
                 writer.Write(TagName);
 
-                // Perf: Avoid allocating enumerator
-                for (var i = 0; i < Attributes.Count; i++)
+                if (_attributes != null)
```

Comment:
Wow, not going for the #PranavPoints? Could have been `for (var i = 0; i < _attributes?.Count ?? 0; i++)`. :trollface: